### PR TITLE
APIGOV-00000 - use the SDK notification package

### PR DESCRIPTION
**Pull request by jcollins-axway**

### CODE REVIEW

File: discovery/pkg/apigee/agent.go [hunk 1]
```diff
@@ -109,9 +102,9 @@ func (a *Agent) registerJobs() error {
 	// create job that starts the subscription manager
 	_, err = jobs.RegisterSingleRunJobWithName(newStartSubscriptionManager(a.apigeeClient, a.apigeeClient.GetDeveloperID), "Start Subscription Manager")
 
-	// create job that registers the api validator
-	_, err = jobs.RegisterSingleRunJobWithName(newRegisterAPIValidatorJob(channels.wgActionChan, a.registerValidator), "Register API Validator")
-
+	// register the api validator job
+	// time.Sleep(a.apigeeClient.GetConfig().Intervals.Portal * 2) // wait 2 portal polling intervals before starting hte api validator job
```
Comment (sbolosan) [inline discovery/pkg/apigee/agent.go:106]: comment? uncomment?

**Review by sbolosan — APPROVED** [review]
I like the refactor...

File: discovery/pkg/apigee/agent.go (hunk 1 above)
Comment (jcollins-axway) [reply discovery/pkg/apigee/agent.go:106]: remove

**Review by sbolosan — APPROVED** [review]
all good...

**Pull request merged**